### ISSUE TITLE
voice activation crashes

### ISSUE THREAD

**Issue by tarqs** [loopuleasa/Dota2-TheCore-Config-Engine#86]
Hi, I'm currently using a modified version of your super compact QWERTG layout (I have remapped a few buttons here and there to reflect my personal preferences) 

But when trying to use the ingame voice chat it freezes the screen for a few seconds for some reason. At times it freezes until I release the button, sometimes just a few seconds. But this is very annoying as I am prone to getting ganked or attacked during that time (I think my character stops moving aswell)

Any ideas?

**Comment by dvinesett** [comment]
Do any errors print to your console? If so, include them

**Comment by tarqs** [comment]
I'll have to play a live game to see it happen again, will post it as soon as I have time to play a live game (tried with solo vs bots, didn't go down)

**Comment by dvinesett** [comment]
http://dev.dota2.com/showthread.php?t=174318
This may not be a problem with the Config Engine. Try using the "Streamlined Push-To-Talk" option in Audio > Voice settings.

**Comment by loopuleasa** [comment]
I like that option.

**Comment by tarqs** [comment]
I actually found that out before checking back with this issue and performed the action above. It is indeed the solution to my problem and was wondering if you could standardize it through cfg files aswell?

**Comment by dvinesett** [comment]
I couldn't find a console command for it. Any help?

**Comment by loopuleasa** [comment]
Yes, I don't know the console command for it either, cause I would love to add it to the settings_game cfg file.

It is a relatively new feature added, afaik.

Someone should do some digging in the console with the "find keyword" command maybe they stumble upon this voice option.
